.travis.yml

```diff
@@ -1,8 +1,41 @@
+---
 language: node_js
 node_js:
-  - "0.10.33"
-services: mongodb
+  - '0.10'
+
+sudo: required
+
+branches:
+  only:
+    - master
+    - develop
+
+git:
+  depth: 10
+
+services:
+  - mongodb
+
+cache:
+  directories:
+    - node_modules
+
 before_install:
-  - "npm install -g npm"
-  - "npm cache clean"
-  - "npm install -g adapt-cli"
+  - npm install -g npm
+  - npm install -g adapt-cli
+
+install:
+  - npm config set spin false
+  - npm install
+
+matrix:
+  fast_finish: true
+
+notifications:
+  email: false
```

Comment:
Under the default configuration, Travis-CI should email the PR author and committers on a failed build. If you do not specify any config for email under notifications, it should default as described here: http://docs.travis-ci.com/user/notifications/#Notifications

> By default, email notifications are sent to the committer and the commit author, if they are members of the repository (that is, they have push or admin permissions for public repositories, or if they have pull, push or admin permissions for private repositories).

Also, under the email notification section: http://docs.travis-ci.com/user/notifications/#How-is-the-build-email-receiver-determined%3F

> By default, a build email is sent to the committer and the author, but only if they have access to the repository the commit was pushed to.